### PR TITLE
Check if the node exists before adding to the list of nodes to sort.

### PR DESCRIPTION
**Pull request by carolhmj**
This is to cover a case that @auberryberry found

You can check this version to see if it doesn't break your GLB 😄

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/13640/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13640/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13640/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/13640/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/13640/merge
https://gui.babylonjs.com/?snapshot=refs/pull/13640/merge
https://nme.babylonjs.com/?snapshot=refs/pull/13640/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/13640/merge#BCU1XR#0

File: packages/dev/core/src/assetContainer.ts [hunk 1]
```diff
@@ -216,7 +216,7 @@ export class AssetContainer extends AbstractScene {
     }
 
     private _addNodeAndDescendantsToList(list: Node[], addedIds: Set<number>, rootNode: Node, predicate?: (entity: any) => boolean) {
-        if ((predicate && !predicate(rootNode)) || addedIds.has(rootNode.uniqueId)) {
+        if (!rootNode || (predicate && !predicate(rootNode)) || addedIds.has(rootNode.uniqueId)) {
```
Comment (Popov72) [inline packages/dev/core/src/assetContainer.ts:219]: If `rootNode` is allowed to be null/undefined, we should update the declaration, something like:
```typescript
private _addNodeAndDescendantsToList(list: Node[], addedIds: Set<number>, rootNode?: Node, predicate?: (entity: any) => boolean)
```

**Review by sebavan — APPROVED** [review]

**Comment by auberryberry** [comment]
It looks like this has regressed in 6.0, I'm seeing the same failure I had before.

**Comment by sebavan** [comment]
@auberryberry could you share back a repro for it ? we ll be glad to address it ASAP

**Comment by auberryberry** [comment]
Yeah ok, let me jazz something up
________________________________

**Comment by auberryberry** [comment]
Ok, Line 437 assumes the node has this method: `.getTotalVertices()` which isn't true for this asset.
https://playground.babylonjs.com/#IBBPFV#11

**Comment by sebavan** [comment]
ohhhhh @carolhmj can you have a look ? 

This PG reproduces the issue https://playground.babylonjs.com/#IBBPFV#11 on the latest version.

**Comment by auberryberry** [comment]
[heart]         Aubrey Elizabeth Stearn reacted to your message:
________________________________

**Comment by carolhmj** [comment]
Ooo thanks for the report, have a PR: https://github.com/BabylonJS/Babylon.js/pull/13929

**Pull request merged**